src/helpers.test.ts

```diff
@@ -39,34 +40,35 @@ test('getBaseUrl', () => {
   expect(PLATFORMS.map((platform) => {
     return getBaseUrl(platform, 'latest');
   })).toEqual([
-    'https://uploader.codecov.io/latest/aarch64/codecov',
-    'https://uploader.codecov.io/latest/alpine/codecov',
-    'https://uploader.codecov.io/latest/linux/codecov',
-    'https://uploader.codecov.io/latest/macos/codecov',
-    'https://uploader.codecov.io/latest/windows/codecov.exe',
+    'https://cli.codecov.io/latest/linux/codecov',
```

Comment:
oh, I know this isn't you, but this is definitely a breaking change. we need `aarch64` and `alpine` builds. cc: @drazisil-codecov

Reply:
@t

> oh, I know this isn't you, but this is definitely a breaking change. we need `aarch64` and `alpine` builds. cc: @drazisil-codecov

@thomasrockhu-codecov   Did you create a new issue for this?

Reply:
https://github.com/codecov/codecov-cli/issues/266